client/login/qr-code-login/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { Card, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import AsyncLoad from 'calypso/components/async-load';
+import Main from 'calypso/components/main';
+import { login } from 'calypso/lib/paths';
+
+import './style.scss';
+
+function QrCodeLoginPlaceholder() {
+	return <div className="qr-code-login__placeholder"></div>;
+}
+
+function QrCodeLoginPage( { locale } ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="qr-code-login">
+			<div className="qr-code-login__form">
+				<h1 className="qr-code-login__heading">{ 'Login via the Mobile App' }</h1>
```

Comment:
Minor note: I think we usually use sentence case for those:

```suggestion
				<h1 className="qr-code-login__heading">{ 'Login via the mobile app' }</h1>
```